src/test/java/org/candlepin/subscriptions/tally/billing/QuantityTest.java

```diff
@@ -109,9 +109,9 @@ void testMetricUnitToMetricUnitPreservesValue() {
   @Test
   void testQuantityFromBillableUsage() {
     var billableUsage = new BillableUsage();
-    billableUsage.setUom(SOCKETS);
+    billableUsage.setMetricId(SOCKETS);
     billableUsage.setProductId("foo");
-    createSubscriptionDefinition(billableUsage.getProductId(), billableUsage.getUom().toString());
+    createSubscriptionDefinition(billableUsage.getProductId(), billableUsage.getMetricId());
```

Comment:
update the `createSubscriptionDefinition` definition to rename the `uom` parameter to `metricId`.